Apps/WebClient/src/ClientApp/src/models/immunizationTimelineEntry.ts

```diff
@@ -35,8 +40,8 @@ class ImmunizationAgentViewModel {
     constructor(model: ImmunizationAgent) {
         this.code = model.code;
         this.name = model.name;
-        this.lotNumber = model.lotNumber;
-        this.productName = model.productName;
+        this.lotNumber = model.lotNumber === "" ? "N/A" : model.lotNumber;
```

Comment:
if the value is null/undefined this check wouldn't work and it would still display as blank

Reply:
can it be null or undefined?

Reply:
from the server side, we always initialize string with string.Empty so I think we are good for this one.